pages/common/mise.md

```diff
@@ -0,0 +1,32 @@
+# mise
+
+> Command-line interface for managing versions of different packages.
+> More information: <https://mise.jdx.dev>.
+
+- List all available plugins:
+
+`mise plugins list-all`
+
+- Install a plugin:
+
+`mise plugins add {{name}}`
+
+- List runtime versions available for install:
+
+`mise ls-remote {{name}}`
+
+- Install a specific version of a package:
+
+`mise install {{name}}@{{version}}`
+
+- Set global version for a package:
+
+`mise use --global {{name}}@{{version}}`
+
+- Set local version for a package:
+
+`mise use {{name}}@{{version}}`
+
+- Set environment variable in config:
```

Comment:
```suggestion
- Set environment variable in configuration:
```
We generally prefer having descriptive description, so I would suggest expanding it here, but if you are trying to imply a path with this example, then you can enclose it inside back ticks.